src/schdispatch_gui.cpp

```diff
@@ -1342,8 +1343,8 @@ struct SchdispatchWindow : GeneralVehicleWindow {
 			default: NOT_REACHED();
 
 			case WID_SCHDISPATCH_ADD: {
-				if (!this->IsScheduleSelected()) break;
 
+				if (!this->IsScheduleSelected()) break;
```

Comment:
Better to avoid changes like this which just move whitespace around, if possible